commit 9f3276bb00abe841e541cd856267a72e723175ad
Author: RahifM <rahifmanjatha372@gmail.com>
Date:   Sat Oct 22 09:53:24 2022 +0000

    Use 'release-keys' for build tag

diff --git a/core/Makefile b/core/Makefile
index b418946347..2f918e278d 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -325,7 +325,7 @@ endif
 # Both of these tags will be removed and replaced with "release-keys"
 # when the target-files is signed in a post-build step.
 ifeq ($(DEFAULT_SYSTEM_DEV_CERTIFICATE),build/make/target/product/security/testkey)
-BUILD_KEYS := test-keys
+BUILD_KEYS := release-keys
 else
 BUILD_KEYS := dev-keys
 endif
