diff --git a/Makefile b/Makefile
index 5f4e5e937298..1a305303f064 100755
--- a/Makefile
+++ b/Makefile
@@ -255,8 +255,8 @@ SUBARCH := $(shell uname -m | sed -e s/i.86/x86/ -e s/x86_64/x86/ \
 # Default value for CROSS_COMPILE is not to prefix executables
 # Note: Some architectures assign CROSS_COMPILE in their arch/*/Makefile
 override ARCH		:= arm64
-override CROSS_COMPILE	:= /home/arter97/arm64-gcc/bin/aarch64-elf-
-override CROSS_COMPILE_ARM32	:= /home/arter97/arm32-gcc/bin/arm-eabi-
+override CROSS_COMPILE	:= $(KERNELDIR)/arm64-gcc/bin/aarch64-elf-
+override CROSS_COMPILE_ARM32	:= $(KERNELDIR)/arm32-gcc/bin/arm-eabi-
 
 # Architecture as present in compile.h
 UTS_MACHINE 	:= $(ARCH)
diff --git a/build_clean.sh b/build_clean.sh
index f368db4d968e..19d2511c0170 100755
--- a/build_clean.sh
+++ b/build_clean.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-git clean -fdx
-git reset --hard
+#git clean -fdx
+#git reset --hard
 
 cp defconfig .config
